jsk_fetch_robot/jsk_fetch_startup/scripts/update_workspace_main.sh

```diff
@@ -85,18 +87,22 @@ catkin config -DCMAKE_BUILD_TYPE=Release
 catkin build --continue-on-failure
 CATKIN_BUILD_RESULT=$?
 # Send mail
-MAIL_BODY=""
+echo "" > $TMP_MAIL_BODY_FILE
 if [ $WSTOOL_UPDATE_RESULT -ne 0 ]; then
-    MAIL_BODY=$MAIL_BODY"Please wstool update workspace manually. "
+    echo "Please wstool update workspace manually.\n" >> $TMP_MAIL_BODY_FILE
 fi
 if [ $ROSDEP_INSTALL_RESULT -ne 0 ]; then
-    MAIL_BODY=$MAIL_BODY"Please install dependencies manually. "
+    echo "Please install dependencies manually.\n" >> $TMP_MAIL_BODY_FILE
 fi
 if [ $CATKIN_BUILD_RESULT -ne 0 ]; then
-    MAIL_BODY=$MAIL_BODY"Please catkin build workspace manually."
+    echo "Please catkin build workspace manually.\n" >> $TMP_MAIL_BODY_FILE
 fi
 set +x
 } 2>&1 | tee $LOGFILE
+
+MAIL_BODY=$(cat $TMP_MAIL_BODY_FILE)
```

Comment:
If possible, I would appreciate it if you could leave a comment on how it became necessary to use a temporary mail body file in this way. Something like... 
```
# MAIL_BODY variable is not set when using tee because of the scope.
# See https://github.com/jsk-ros-pkg/jsk_robot/issues/1569
```

Reply:
OK, I'll add comment.